test/Strings/rlexe.xml

```diff
@@ -268,4 +268,10 @@
       <compile-flags>-args summary -endargs</compile-flags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>constructorConcat.js</files>
+      <compile-flags>-lic:1 -off:backend -mic:1 -bgjit-</compile-flags>
```

Comment:
>-off:backend [](start = 28, length = 12)

what does off: backend will do? run in interpreted mode?

Reply:
-off:backend stops function JITing but not loop JITing

Reply:
This might be a better test case as it does not rely on JIT loop body:


function f(arg)
{
    var i = 0;
    while (i < 5) {
        i++;
        arg += "this_should_not_stay";
    }
}
f("Hello");
f(Int8Array);

if(!Int8Array.toString().includes("this_should_not_stay"))
{
    WScript.Echo("Passed");
}
else
{
    WScript.Echo("FAILED");
}


---
In reply to: [283916360](https://github.com/microsoft/ChakraCore/pull/6121#discussion_r283916360) [](ancestors = 283916360)